tests/infra/node.py

```diff
@@ -347,12 +350,16 @@ def client(self, identity=None, signing_identity=None, **kwargs):
         return ccf.clients.client(self.pubhost, self.pubport, **akwargs)
 
     def suspend(self):
-        self.remote.suspend()
-        LOG.info(f"Node {self.node_id} suspended...")
+        if not self.suspended:
```

Comment:
I think we want to raise here, this is test infra, we don't want it silently not suspending or resuming.

Reply:
Added asserts.